centraldogma/src/main/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplier.java

```diff
@@ -225,12 +252,24 @@ private static long remainingTime(long totalTime, long startedTime) {
     }
 
     // visible for testing
-    static JsonNode defaultProperties() {
-        final ObjectNode properties = objectMapper.createObjectNode();
+    static JsonNode convertPropertyListToJsonNode(List<Property<?>> properties) {
+        final ObjectNode propertiesObjectNode = objectMapper.createObjectNode();
+        properties.forEach(
+                property -> {
+                    propertiesObjectNode.set(
+                            property.definition().name(),
+                            objectMapper.valueToTree(property.value())
+                    );
+                }
+        );
+        return propertiesObjectNode;
+    }
+
+    // visible for testing
+    static List<Property<?>> defaultProperties() {
+        List<Property<?>> properties = new ArrayList<>();
         ProcessorProperties.PROPERTY_DEFINITIONS
-                .forEach(definition -> properties.set(definition.name(),
-                                                      objectMapper.valueToTree(definition.defaultValue()))
-                );
+                .forEach(definition -> properties.add(new DynamicProperty(definition)));
```

Comment:
There's no point to use `DynamicProperty` here. It's for modifiable property instance, so let's use `Property.ofStatic()` instead.

Reply:
https://github.com/line/decaton/pull/134/commits/6d9e69e9a99e3e52ca4ece5f2c1629dcce846497